### ISSUE TITLE
Adding Location data to Loughborough

### ISSUE THREAD

**Issue by polly64** [colouring-cities/colouring-london#166]
@MKIM1008 @Taimaz22

- [ ] could you possible go through all categories for Location data and see what is  available for Loughborough? and then access as many as possible. Ask me for a copy the excel file on data categories which will be useful for when we meet and you can tick off ones you can find- if i haven't given it yet

- [ ] could you  ask if council have an open addressbase? If not can we access the council's  behind a fire wall? - 

- [ ]  need to be able to search Loughborough by postcode

**Comment by MKIM1008** [comment]
@polly64 @taimaz22 

I will reach out to the council for access to the open address data with the enquiry about the building demolition and construction history.

**Comment by MKIM1008** [comment]
@polly64 

Yes, please share the Excel file for Location data with me to tick off the available data from me. I will remind you of the file again during the meeting tomorrow.

**Comment by polly64** [comment]
done

**Comment by polly64** [comment]
@MKIM1008 can you move the information on location which you just put in land use to location here , thanks and then lets use this section for all location data issues

**Comment by MKIM1008** [comment]
@polly64 

Charnwood Borough Council - information from their Land Use data

![image](https://github.com/colouring-cities/colouring-london/assets/88775529/386a82bf-de19-40ec-8ec8-1554f7be5ed1)

**Comment by polly64** [comment]
@MKIM1008 thanks. @matkoniecz we are just seeing which of these fields we are permitted to include

**Comment by polly64** [comment]
> @polly64 @taimaz22
> 
> I will reach out to the council for access to the open address data with the enquiry about the building demolition and construction history / geology data.

@MKIM1008 can you create a new issue for this called 'Colouring Loughborough Digital twin 1: simulating change'

**Comment by polly64** [comment]
@MKIM1008 key ones we want are building number and name and street, if we can't have these as open data can we show these in locked fields so data can't be downloaded ?

**Comment by MKIM1008** [comment]
@polly64 

I will get a confirmation on this from the council.

**Comment by matkoniecz** [comment]
I marked postcode task as done some time ago.
